fulcio.proto

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+package dev.sigstore.fulcio.v1;
+
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+option go_package = "github.com/sigstore/fulcio/pkg/generated/protobuf";
+
+service CA {
+    /**
+     * Returns information about the current state of the transparency log
+     */
+    rpc GetSigningCert(CertificateRequest) returns (CertificateResponse){
+        option (google.api.http) = {
+          post: "/api/v1/signingCert"
+          body: "*"
+        };
+    }
```

Comment:
Can we document some of the custom response munging we have to do on the HTTP path?  I could see folks naively trying to reimplement this proto API and having a bad time.

Reply:
I could resurrect the old `openapi.yaml` doc and make sure it is up to date

Reply:
Oh, I was just talking about in the `rpc` comment.  Pretty sure you can generate swagger docs from these as well, although if I've done it it's been years...